src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/ViewRenderer.js

```diff
@@ -34,7 +35,7 @@ export default class ViewRenderer extends React.Component<Props> {
         return rerenderAttributeValues.join('__');
     };
 
-    renderView(route: Route, child: Element<*> | null = null) {
+    renderView(route: Route, child: Element<*> | null = null, hasSidebar: boolean = false) {
```

Comment:
Having this flag on this method feels strange... Why is it needed? The toolbar is also not mentioned in this method...

Reply:
because we only want to clear the sidebarstore if it was present before and not there afterwards - to avoid rerendering sidebar